qa-tests-backend/src/test/groovy/NetworkFlowTest.groovy

```diff
@@ -704,6 +703,9 @@ class NetworkFlowTest extends BaseSpecification {
                 def sourceNamespacesFromNetworkPolicy = ingressNamespaceSelectors.collect {
                     it."namespaceSelector"."matchLabels"."namespace.metadata.stackrox.io/name"
                 }
+                sourceNamespacesFromNetworkPolicy.addAll(ingressNamespaceSelectors.collect {
+                    it."namespaceSelector"."matchLabels"."kubernetes.io/metadata.name"
+                })
                 assert sourceDeploymentsFromNetworkPolicy.sort() == sourceDeploymentsFromGraph.sort()
                 if (!deployedNamespaces.containsAll(sourceNamespacesFromNetworkPolicy)) {
```

Comment:
not touched in this PR, but do you know why we didn't see the prints below when this was true when the test was failing?

Reply:
I saw these prints

Reply:
perhaps I missed them before, then